app/styles/_core.less

```diff
@@ -79,6 +79,11 @@
 
 /* Fix some styles with the angular-ui-select bootstrap theme that isn't right when combined with all of our styles */
 .ui-select-bootstrap {
+  /* Fixes ui-select's input box extending to 100% page width if the page has been resized after page-load */
```

Comment:
I'd add a link to the GitHub issue as well. Then we know when the issue is fixed and can take out the workaround.